WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -23,7 +24,10 @@ @implementation XCUIElement (FBIsVisible)
 
 - (BOOL)fb_isVisible
 {
-  return self.fb_lastSnapshot.fb_isVisible;
+  if (SYSTEM_VERSION_LESS_THAN(@"11.0") || [FBConfiguration shouldUseTestManagerForVisibilityDetection]) {
```

Comment:
Can you remove `SYSTEM_VERSION_LESS_THAN` check? If that work it should work current iOS version and pass green on current tests.